### PR TITLE
Bug 1874895: Enabling use of $BRIDGE_E2E_BROWSER_NAME with Cypress

### PR DESCRIPTION
**Pull request by miyamotoh**
Like we did with Protractor, this PR will enable the (re)use of `$BRIDGE_E2E_BROWSER_NAME` with Cypress.

Signed-off-by: Hiro Miyamoto <miyamotoh@us.ibm.com>

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "test-gui": "yarn run test-suite --suite all",
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
     "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
-    "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser chrome --headless",
+    "test-cypress-headless": "test -z \"${BRIDGE_E2E_BROWSER_NAME}\" && BRIDGE_E2E_BROWSER_NAME=chrome ; cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME} --headless",
```
Comment (spadgett) [inline frontend/package.json:30]: Does this work?

```suggestion
    "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
```

Comment (miyamotoh) [reply frontend/package.json]: That's slick, and works! 👍

File: frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts [hunk 2]
```diff
@@ -35,7 +35,7 @@ const launcherLink = {
     namespaceDashboard: {
       namespaces: [testName],
     },
-    text: 'Help Menu Link',
+    text: 'Namespace Dashboard Link',
```
Comment (spadgett) [inline frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts:38]: This change looks unrelated

Comment (miyamotoh) [reply frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts:38]: Sorry, screwed up with `--amend` while trying to squash local commits. Starting over..

Comment (miyamotoh) [reply frontend/integration-tests/tests/dashboards/project-dashboard.scenario.ts]: Okay, commits squashed.

**Comment by miyamotoh** [comment]
Checks passed. Is this good for merge, @spadgett ?
